### PR TITLE
mkcloud: add support for deploying octavia

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -3306,7 +3317,7 @@ function onadmin_proposal
     safely oncontroller check_crm_failcounts
     # For all remaining proposals, check for HA failures after each deployment
     for proposal in horizon ceilometer heat manila trove \
-        designate barbican magnum sahara aodh tempest; do
+        designate octavia barbican magnum sahara aodh tempest; do
```
Comment (ilausuch) [inline scripts/qa_crowbarsetup.sh]: octavia depends on barbican. Is the order important here?

Comment (dirkmueller) [reply scripts/qa_crowbarsetup.sh]: good point, fixed.

File: scripts/lib/mkcloud-common.sh [hunk 2]
```diff
@@ -730,6 +730,7 @@ fi
 : ${want_mtu_size:=1500}
 # proposals:
 : ${want_designate_proposal:=0}
+: ${want_octavia_proposal:=0}
```
Comment (bmwiedemann) [inline scripts/lib/mkcloud-common.sh]: minor nit: move it down 2 lines to keep the entries sorted.

File: scripts/mkcloud [hunk 3]
```diff
@@ -1284,6 +1284,7 @@ Optional
         The following proposals are default off:
         Set to 1 to enable \$PROPOSAL to be deployed. e.g.
         want_designate_proposal=1 (Cloud9+ only)
+        want_octavia_proposal=1 (Cloud9+ only)
```
Comment (bmwiedemann) [inline scripts/mkcloud]: maybe also move down to keep it sorted?

File: scripts/lib/mkcloud-common.sh (hunk 2 above)
Comment (dirkmueller) [reply scripts/lib/mkcloud-common.sh]: done

File: scripts/mkcloud (hunk 3 above)
Comment (dirkmueller) [reply scripts/mkcloud]: done

**Pull request merged**